packages/property-grid/package.json

```diff
@@ -52,11 +52,11 @@
     "@bentley/itwin-client": "^2.6.0",
     "@bentley/orbitgt-core": "^2.6.0",
     "@bentley/presentation-common": "^2.6.0",
-    "@bentley/presentation-components": "^2.6.0",
+    "@bentley/presentation-components": "^2.16.2",
```

Comment:
did you need to bump to 2.16.2? whats the lowest core ver your changes are compatible with?